helm/botkube/templates/clusterrole.yaml

```diff
@@ -21,4 +21,18 @@ rules:
   - apiGroups: [""]
     resources: ["namespaces"]
     verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Now it's under flag `.Values.rbac.create`, which should retain old behaviour as  the default.
When ready to release, we should add a new feature flag for RBAC.